packages/zipkin-instrumentation-hapi/src/hapiMiddleware.js

```diff
@@ -93,10 +93,11 @@ exports.register = (server, {tracer, serviceName = 'unknown', port = 0}, next) =
 
   server.ext('onPreResponse', (request, reply) => {
     const {response} = request;
+    const statusCode = response.isBoom ? response.output.statusCode : response.statusCode;
```

Comment:
Interesting, is this usually done for all errors (client errors as well as server errors)?